### PR TITLE
Add Augur 8.0.0

### PR DESCRIPTION
**Pull request by garfinjm**
[Documentation for 8.0.0 changes](https://docs.nextstrain.org/projects/augur/en/stable/releases/changelog.html)

Help displays correctly in docker:
```
[garfin@dev-env docker-builds]$ docker run garfinjm/augur:8.0.0 augur --help
usage: augur [-h]
             {parse,filter,mask,align,tree,refine,ancestral,translate,reconstruct-sequences,clades,traits,sequence-traits,lbi,distance,titers,frequencies,export,validate,version,import}
             ...

Augur: A bioinformatics toolkit for phylogenetic analysis.

positional arguments:
  {parse,filter,mask,align,tree,refine,ancestral,translate,reconstruct-sequences,clades,traits,sequence-traits,lbi,distance,titers,frequencies,export,validate,version,import}
    parse               Parse delimited fields from FASTA sequence names into
                        a TSV and FASTA file.
    filter              Filter and subsample a sequence set.
    mask                Mask specified sites from a VCF or FASTA file.
    align               Align multiple sequences from FASTA.
    tree                Build a tree using a variety of methods.
    refine              Refine an initial tree using sequence metadata.
    ancestral           Infer ancestral sequences based on a tree.
    translate           Translate gene regions from nucleotides to amino
                        acids.
    reconstruct-sequences
                        Reconstruct alignments from mutations inferred on the
                        tree
    clades              Assign clades to nodes in a tree based on amino-acid
                        or nucleotide signatures.
    traits              Infer ancestral traits based on a tree.
    sequence-traits     Annotate sequences based on amino-acid or nucleotide
                        signatures.
    lbi                 Calculate LBI for a given tree and one or more sets of
                        parameters.
    distance            Calculate the distance between sequences across entire
                        genes or at a predefined subset of sites.
    titers              Annotate a tree with actual and inferred titer
                        measurements.
    frequencies         infer frequencies of mutations or clades
    export              Export JSON files suitable for visualization with
                        auspice.
    validate            Validate files related to augur consumption or export.
    version             Print the version of augur.
    import              Import analyses into augur pipeline from other systems

optional arguments:
  -h, --help            show this help message and exit

```
Tested and working as a drop-in replacement for 7.2.0 in [augur-nf](https://github.com/AbigailShockey/augur-nf) using Singularity.

```
#augur-nf user config
docker.enabled = false
singularity.enabled = true
process.stageInMode = 'copy'
process.container = 'garfinjm/augur:8.0.0'
```

```
N E X T F L O W  ~  version 19.10.0
Launching `/home/mdh/garf0012/nextstrain/augur-nf/augur.nf` [curious_mcnulty] - revision: 7f085e94a9
[57/d406df] process > tree (1)      [100%] 1 of 1, cached: 1 ✔
[c3/c11ac8] process > refine (1)    [100%] 1 of 1 ✔
[b7/5811aa] process > ancestral (1) [100%] 1 of 1 ✔
[6b/6e91e7] process > translate (1) [100%] 1 of 1 ✔
[f2/9ab787] process > traits (1)    [100%] 1 of 1 ✔
[2f/59af7a] process > export (1)    [100%] 1 of 1 ✔
Completed at: 15-Jun-2020 14:21:56
Duration    : 3m 47s
CPU hours   : 0.1 (14.6% cached)
Succeeded   : 5
Cached      : 2
```

### CODE REVIEW

**Comment by kapsakcj** [comment]
beautiful! Thanks for the new version and for testing!

File: augur/8.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL container.version="1"
```
Comment (kapsakcj) [inline augur/8.0.0/Dockerfile:5]: Please change to `LABEL dockerfile.version="1"`. See #118 for why

**Comment by kapsakcj** [comment]
it's probably ok as I think docker and singularity will create a directory if you mount/bind a volume inside the container....but there's no `mkdir /data` anywhere in the dockerfile. Just want to document in case someone hits an issue with this.

**Comment by kapsakcj** [comment]
and ignore the travis build failling...there is something going on with the ariba test

File: augur/8.0.0/Dockerfile (hunk 1 above)
Comment (garfinjm) [reply augur/8.0.0/Dockerfile]: Patched in `/data` and `dockerfile.version`. Thanks for the review! 

I've been meaning to make a checklist of all the little things that are easy to forget (locale, /data, update Program_Licenses.md, etc.)

**Pull request merged**